### DISCUSSION TITLE
Filter layout control

### DISCUSSION THREAD

**Discussion by brandonzylstra** [avo-hq/avo#1840] [Feature requests]
I thought I saw something in the documentation showing a way to control the layout of the filter menu, but now that I'm looking for it I can't find it.  

Here's my situation: when on a laptop screen with the browser zoomed in to "Actual Size" (in Safari's terminology), my filters run off the screen, which seems sub-optimal:

<img width="358" alt="image" src="https://github.com/avo-hq/avo/assets/9854/e867a47e-a9d2-4d22-9ef5-faeba187b04a">

I thought I saw something that laid them out horizontally, but that may or may not solve the problem for me.

Here are a couple of other possible solutions:
- the ability to create a partial (e.g. `filter_layout.html.erb`) that controlled the layout.
- the ability to create multiple filter menu buttons (e.g. "Filter by Compliance", "Filter by Job Status", and "Filter by Org Chart")

This is not a super high priority.  I can live with scrolling the filters, but I think being able to control the layout could be very helpful to me and many others.

**Comment by adrianthedev** [comment]
Hmmm, I can see the scrolling issue.
Now, I'm thinking, if we enable the horizontal layout and you have too many... you'll still have to scroll.

Please give us some time to launch Avo 3 and revisit this after. The new filters might cure some of these issues.
At some point we'd like to merge these filters into the other ones somehow...

Thanks for bringing this up.
